config/fcos/v1_6_exp/validate_test.go

```diff
@@ -237,6 +237,141 @@ func TestValidateGrubUser(t *testing.T) {
 	}
 }
 
+func TestValidateMountPoints(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid config (has prefix "/etc" or "/var")
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/etc/foo"),
+							},
+							{
+								Path: util.StrToPtr("/var"),
+							},
+						},
+					},
+				},
+			},
+		},
+		// invalid config (path name is '/')
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/"),
+							},
+						},
+					},
+				},
+			},
+		},
+		// invalid config (path is /boot)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path:          util.StrToPtr("/boot"),
+								WithMountUnit: util.BoolToPtr(true),
+							},
+						},
+					},
+				},
+			},
+
+			out:     common.ErrMountPointForbidden,
+			errPath: path.New("yaml", "storage", "filesystems", 0, "path"),
+		},
+		// valid config (path is nil)
```

Comment:
I think this can be dropped in favor of the expanded "valid" test case above.  As written, this one isn't especially useful, since it has *two* conditions that would cause us not to warn.

Reply:
No probkem, made a separate commit with the dropped test

Reply:
Got confused about what part needed changing thanks for specifying 
Dropped //valid config (path is nil)